### DISCUSSION TITLE
A Second Box

### DISCUSSION THREAD

**Discussion by oldgristmill** [mcguirepr89/BirdNET-Pi#941] [Ideas]
I've got a pretty big property and would love to have a second Pi listening in another area, but ideally both sets of detections would work their way into a single set of stats. What would be amazing would be for there to be some sort of master/slave relationship between Birdnet-pi instances so that either one streams audio to the other and then both sets of detections happen on a single Pi, or instead somehow only the results are passed along to the database on one Pi.

Maybe there's already a way to do this by adding RTSP streams from both the local box and another somehow, but it would be great if all I had to do was install the same software on both boxes and just configure them to work together.

**Comment by srchild** [comment]
That could be a good idea. But let me offer an alternative idea for if you have two installations.

Two ears can offer stereo hearing! My first installation last year was facing West towards moorland and farmland. I was aware that it was missing birds which I knew were present to the East in different terrain, notably a stream with some ponds and small trees. This year I added a second installation facing East and this has been interesting in a couple of ways.

One bird I knew I was missing last year was the grasshopper warbler (only 16 detections last year, 31 this year) My east-facing installation has recorded 506 detections this year (figures based on the 15minute stats)

Being moorland with few trees we rarely hear Tawny owls and when we do hear them live with our ears they are usually from trees about half a mile to the East so I assumed that is where last year's Tawny owl records came from. This year we have been hearing tawny owl a little more often, but my second installation facing east (same hardware and same microphone/soundcard) has rarely recorded Tawny when my original west-facing installation has done - i.e. this year the Tawny are probably calling from the few trees about half a mile to the West instead of the easterly trees. Extra information from a stereo installation which would be lost if they were merged into one database.
